Program.cs

```diff
@@ -14,7 +14,22 @@ class Program
   {
     static void Main(string[] args)
     {
-      MainAsync().GetAwaiter().GetResult();
+      Database.Init_Database();
+      Database.defaultSetup();
+      var tmp = Database.runSQL("SELECT * FROM CustomRoles");
+
+      foreach (var i in tmp)
+      {
+        foreach (var j in i)
+        {
+          Console.WriteLine(j);
+        }
+      }
+
+
+      //MainAsync().GetAwaiter().GetResult();
+      //GuildmemberTask().GetAwaiter().GetResult();
```

Comment:
Auskommentiert = Bot funktioniert nicht

```suggestion
      MainAsync().GetAwaiter().GetResult();
```